pkgs/tools/package-management/rpm/default.nix

```diff
@@ -28,14 +28,24 @@ stdenv.mkDerivation rec {
     "--with-external-db"
     "--with-lua"
     "--enable-python"
+    "--enable-ndb"
     "--localstatedir=/var"
     "--sharedstatedir=/com"
   ];
 
-  postPatch = ''
-    # For Python3, the original expression evaluates as 'python3.4' but we want 'python3.4m' here
-    substituteInPlace configure.ac --replace 'python''${PYTHON_VERSION}' ${python.executable}
```

Comment:
This didn't apply cleanly anymore.
I am not 100% sure WHY we wanted 3.4m here, but it seems to work fine without this line.

@copumpkin any idea? :)

Reply:
might be resolved now.

Reply:
Does build python bindings and do they work?

Reply:
Hmmm... I got them to work with dnf/libdnf previously, but I switched things over to microdnf (https://github.com/NixOS/nixpkgs/pull/107348) 

I guess this would be an ok test?

```
mseeger@mseeger-mbp store % ./px24kgsz8wpnfs0zv5zg28l66myzw3ki-python3-3.8.6/bin/python3.8
Python 3.8.6 (default, Nov 26 2020, 00:23:22) 
[Clang 7.1.0 (tags/RELEASE_710/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys; sys.path.insert(0, "/nix/store/2ws7pc2n2578qxjx5802v43kfnvsf0k1-rpm-4.16.1.2/lib/python3.8/site-packages/"); import rpm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/2ws7pc2n2578qxjx5802v43kfnvsf0k1-rpm-4.16.1.2/lib/python3.8/site-packages/rpm/__init__.py", line 38, in <module>
    from rpm._rpm import *
ImportError: dlopen(/nix/store/2ws7pc2n2578qxjx5802v43kfnvsf0k1-rpm-4.16.1.2/lib/python3.8/site-packages/rpm/_rpm.so, 2): Symbol not found: _lua_strlen
  Referenced from: /nix/store/2ws7pc2n2578qxjx5802v43kfnvsf0k1-rpm-4.16.1.2/lib/librpmio.9.dylib
  Expected in: flat namespace
 in /nix/store/2ws7pc2n2578qxjx5802v43kfnvsf0k1-rpm-4.16.1.2/lib/librpmio.9.dylib
 ```

Reply:
Actually, no.

That also doesn't work in the old one:

```
>>> import sys; sys.path.insert(0, "/nix/store/d6xiz1csn529pqbmrxbvs3c7g2sv06xb-rpm-4.15.1/lib/python3.8/site-packages"); import rpm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/d6xiz1csn529pqbmrxbvs3c7g2sv06xb-rpm-4.15.1/lib/python3.8/site-packages/rpm/__init__.py", line 38, in <module>
    from rpm._rpm import *
ImportError: dlopen(/nix/store/d6xiz1csn529pqbmrxbvs3c7g2sv06xb-rpm-4.15.1/lib/python3.8/site-packages/rpm/_rpm.so, 2): Symbol not found: _lua_strlen
  Referenced from: /nix/store/d6xiz1csn529pqbmrxbvs3c7g2sv06xb-rpm-4.15.1/lib/librpmio.9.dylib
  Expected in: flat namespace
 in /nix/store/d6xiz1csn529pqbmrxbvs3c7g2sv06xb-rpm-4.15.1/lib/librpmio.9.dylib
 ```

Reply:
Ah... this problem actually solved itself in that we don't have a 'm' suffix anymore :)

https://docs.python.org/3/whatsnew/3.8.html#build-and-c-api-changes

```
Only python3.8 program is installed, python3.8m program is gone.

Only python3.8-config script is installed, python3.8m-config script is gone.

The m flag has been removed from the suffix of dynamic library filenames: extension modules in the standard library as well as those produced and installed by third-party packages, like those downloaded from PyPI. On Linux, for example, the Python 3.7 suffix .cpython-37m-x86_64-linux-gnu.so became .cpython-38-x86_64-linux-gnu.so in Python 3.8.
```